
# The parameter is incorrect while using sns.pairplot()

So I currently have a dataset that is 122X30 named df.
After reading it in like so:
df = pd.read_csv('data.csv')

I wanted to get a general idea of what the relationship of the features so I decided to use seaborn's pairplot like so:
import matplotlib.pyplot as plt
import seaborn as sns
sns.pairplot(df)
plt.show()

Unfortunately, however, I get this:

I've tried changing matplotlib's backend to Augg and other bankends but to no avail.
Any idea what is going on?

        